phpBB/styles/prosilver/template/forumlist_body.html

```diff
@@ -34,18 +34,18 @@
 			<dl class="row-item">
```

Comment:
You should remove `FORUM_IMG_STYLE` template variable

Reply:
I would still keep it for now since some styles or extensions might still be using it.